shared-bindings/adafruit_bus_device/spi_device/SPIDevice.c

```diff
@@ -89,6 +89,10 @@ STATIC mp_obj_t adafruit_bus_device_spidevice_make_new(const mp_obj_type_t *type
 
     busio_spi_obj_t *spi = args[ARG_spi].u_obj;
 
+    if (!mp_obj_is_type(args[ARG_chip_select].u_obj, &digitalio_digitalinout_type)) {
+        mp_raise_TypeError_varg(translate("unsupported %q type"), digitalio_digitalinout_type.name);
+    }
```

Comment:
We now have more validation helper routines that reduce the boilerplate and use a standard error message:
https://github.com/adafruit/circuitpython/blob/e0cb8ef17e294a502d3311681230e1c36ca0ce5f/py/argcheck.c#L233

```suggestion
    mp_arg_validate_type(ARG_chip_select].u_obj, &digitalio_digitalinout_type, MP_QSTR_chip_select);
```

Reply:
Sorry I knew there were new ones and thought I had it. Apparently I did not!

Reply:
Could you retest the validation in the REPL just to make sure I did not typo the fix in some way? Thanks.

Reply:
I built it locally and tested it before I pushed it from my own machine. I caught the typo.

Reply:
Great, thanks!